pkg/provider/provider.go

```diff
@@ -80,6 +80,17 @@ func DeploymentsParse(deploymentFiles []string, deploymentVars map[string]string
 		}
 	}
 
+	for k, v := range deploymentVars {
+		_, err := os.Stat(v)
```

Comment:
why not read the file directly like in:
https://github.com/prometheus/prombench/blob/b8b29233ce43946a6e13407dc461025ff42a1966/pkg/provider/gke/gke.go#L75

Reply:
I will update it